manager/testdata/new-blueprints/scenario5/arrow-flight-module.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: M4DModule
+metadata:
+  name: arrow-flight-write
+  namespace: m4d-system
+  labels:
+    version: 0.0.1
+    name: arrow-flight-write
+spec:
+  description: "This module deploys an arrow-flight service"
+  type: service   # deployed by fybrik
+  chart:
+    name: ghcr.io/mesh-for-data/write-module:0.1.0
+    values: {}
+  statusIndicators:
+  - kind: Deployment
+    successCondition: status == SUCCEEDED # TBD with correct values
```

Comment:
We don't need to provide a status indicator for standard resources, like Deployment.